documentation/templates/bolt_configuration_reference.md.erb

```diff
@@ -0,0 +1,61 @@
+# `bolt.yaml` options
```

Comment:
```suggestion
# DEPRECATED: `bolt.yaml` options
```
Let's see what this looks like in preview 😄  I've been doing some digging and some of our other products do this: https://puppet.com/docs/continuous-delivery/3.x/continuous_delivery_agent.html#continuous_delivery_agent

Reply:
Updated the preview site. :)

Reply:
I don't hate it